src/main/kotlin/DetektDemo.kt

```diff
@@ -1 +1,51 @@
-class DetektDemo {}
+class DetektDemo {
+    private   val unusedVariable :   String = "unused"
+
+    fun isHTTPSuccess(statusCode: Int): Boolean = statusCode == 200
+
+    fun veryLongFunction(){
+        println("Hello")
+        println("Hello")
+        println("Hello")
+        println("Hello");
+        println("Hello")
+        println("Hello")
+        println("Hello")
+        println("Hello");
+        println("Hello")
+        println("Hello")
+    }
+
+    fun fixMyStyle(x:Int)   : Int {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tokgozmusa/detekt-danger/blob/b1db8249b3b7765813bc2400c54d7c55f4c41ab9/src/main/kotlin/DetektDemo.kt#L19"></span>The function fixMyStyle is too long. The maximum length is 10.</td>
    </tr>
  </tbody>
</table>